pyaedt/icepak.py

```diff
@@ -5741,3 +5750,320 @@ def _assign_blower(self, props, faces, inlet_face, fan_curve_flow_unit, fan_curv
+            props["LowSide"] = {"Radiate": False}
+        else:
+            props["LowSide"] = {"Radiate": True,
+                                "RadiateTo": "AllObjects",
+                                "Surface Material": low_side_rad_material}
+        if high_side_rad_material is not None:
+            props["LowSide"] = {"Radiate": False}
+        else:
+            props["HighSide"] = {"Radiate": True,
+                                 "RadiateTo - High": "AllObjects - High",
+                                 "Surface Material - High": high_side_rad_material}
+        props["Shell Conduction"] = shell_conduction
+        if not boundary_name:
+            boundary_name = generate_unique_name("Plate")
+        bound = BoundaryObject(self, boundary_name, props, "Conducting Plate")
+        return _create_boundary(bound)
+
+    def assign_conducting_plate_with_thickness(self, obj_plate, boundary_name=None, total_power="0W",
+                                               thickness="1mm", solid_material="Al-Extruded",
+                                               shell_conduction=False, low_side_rad_material=None,
+                                               high_side_rad_material=None):
+        """
+        Assign thermal boundary conditions with thickness specification to a conducting plate.
+
+        Parameters
+        ----------
+        obj_plate : str or int or list
+            The object to which the boundary is assigned. Surface name
+            if str, face id if int.
+        boundary_name : str, optional
+            Boundary name. If not provided, a name is generated
+            automatically. Default is ``None``
+        total_power : str or float or dict, optional
+            Power dissipated by the plate. If float, the default unit is
+            ``"W"``. A transient or temperature dependent power can be
+            assigned with a dictionary. Default is ``"0W"``.
+        thermal_resistance : str or float, optional
+            If ``"Thermal Resistance"`` is specified as the thermal
+            condition, this parameter represents the thermal resistance
+            of the plate. If float, the default unit is ``"Kel_per_W"``.
+            Default is ``"0Kel_per_W"``.
+        solid_material : str, optional
+            If ``"Thickness"`` is specified as the thermal condition,
+            this parameter represents the material of the conducting plate.
+            Default is ``"Al-Extruded"``.
+        shell_conduction : bool, optional
+            Indicates whether shell conduction is considered.
+            Default is ``False``.
+        low_side_rad_material : str, optional
+            Material on the low side for radiation. If ``None``,
+            radiation is disabled on the low side. Default is ``None``.
+        high_side_rad_material : str, optional
+            Material on the high side for radiation. If ``None``,
+            radiation is disabled on the high side. Default is ``None``.
```

Comment:
```suggestion
        obj_plate : str or int or list
            Object to assign the boundary to. If a string, specify a surface name.
            If an integer, specify a face ID.
        boundary_name : str, optional
            Boundary name. The default is ``None``, in which case a name is generated
            automatically.
        total_power : str or float or dict, optional
            Power dissipated by the plate. The default is ``"0W"``. If a float, the
            default unit is ``"W"``. A transient or temperature-dependent power
            can be assigned with a dictionary. 
        thermal_resistance : str or float, optional
            If ``thermal_specification="Thermal Resistance"``, this parameter
            represents the thermal resistance of the plate. The default is ``"0Kel_per_W"``.
            If a float, the default unit is ``"Kel_per_W"``.
        solid_material : str, optional
             If ``thermal_specification="Thickness"``, this parameter represents the
             material of the conducting plate. The default is ``"Al-Extruded"``.
        shell_conduction : bool, optional
            Whether to consider shell conduction. The default is ``False``.
        low_side_rad_material : str, optional
            Material on the low side for radiation. The default is ``None``,
            in which case radiation is disabled on the low side.
        high_side_rad_material : str, optional
            Material on the high side for radiation. The default is ``None``,
            in which case radiation is disabled on the high side.
```
These parameter definitions here don't match what's in the function definitoni. For example, ``thermal_specification`` does not appear in the method definition, but ``thickness`` does.